apps/sms/views/conversation/style/attachment.css

```diff
@@ -127,15 +125,23 @@
   overflow: hidden;
   text-overflow: ellipsis;
 }
+/* .file-name has a dir="auto". Moreover, sms.css in not included in attachment
```

Comment:
For the `text-align` changes that follows, it is not exactly equivalent to before, but I think it is more consistent now.

**Before patch**
![conversation_attachment_01](https://cloud.githubusercontent.com/assets/1576598/10478095/609c9c7a-725b-11e5-905c-0dfeee24977a.png)

**After**
![conversation_attachment_01](https://cloud.githubusercontent.com/assets/1576598/10478093/5b5f4f6e-725b-11e5-9ecd-c2c77431c1e6.png)

(Please don't pay attention to the phone icon, it will be fixed in the bug for gaia-icons)